From: Danit Goldberg <danitg@mellanox.com>
Date: Wed, 6 May 2020 10:46:54 +0300
Subject: RDMA/cm: Remove unused store to ret in cm_rej_handler
Patch-mainline: v5.8-rc1
Git-commit: 42113eed8f10533ba419df535b6c0ff9141a948c
References: jsc#SLE-15176

The 'goto out' label doesn't read ret, so don't set it.

Link: https://lore.kernel.org/r/20200506074701.9775-4-leon@kernel.org
Signed-off-by: Danit Goldberg <danitg@mellanox.com>
Signed-off-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Jason Gunthorpe <jgg@mellanox.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/core/cm.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/infiniband/core/cm.c
+++ b/drivers/infiniband/core/cm.c
@@ -3058,7 +3058,6 @@ static int cm_rej_handler(struct cm_work
 			 __func__, be32_to_cpu(cm_id_priv->id.local_id),
 			 cm_id_priv->id.state);
 		spin_unlock_irq(&cm_id_priv->lock);
-		ret = -EINVAL;
 		goto out;
 	}
 
